server/discussion/permissions.js

```diff
@@ -1,55 +1,61 @@
-import { Discussion, Pub, CommunityAdmin, PubManager } from '../models';
-import { checkIfSuperAdmin } from '../utils';
+import { Discussion, Pub, CommunityAdmin, PubManager, Branch, BranchPermission } from '../models';
+import { getBranchAccess } from '../branch/permissions';
 
-export const getPermissions = ({ discussionId, pubId, userId, communityId }) => {
+const userEditableFields = ['title', 'content', 'text', 'isArchived', 'highlights', 'labels'];
+
+export const getPermissions = async ({
+	branchId,
+	discussionId,
+	pubId,
+	userId,
+	communityId,
+	discussHash,
+}) => {
 	if (!userId) {
 		return new Promise((resolve) => {
```

Comment:
Because every async function returns a promise - can't we simply say `return {};` here?

Reply:
Yeah, you're right, that's much better.